From 8798d892d9ad6b149b553660fab785bf6b25f577 Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Fri, 16 Nov 2012 10:59:38 -0500
Subject: [PATCH 0047/1936] PadNoteTest: Relaxing constraint

One output sample varies slightly with different optimizations.
It is not outside of a realistically expected deviation, so the bounds are
now relaxed to let the test pass.
---
 src/Tests/PadNoteTest.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Tests/PadNoteTest.h b/src/Tests/PadNoteTest.h
index ee93827f..5e202b36 100644
--- a/src/Tests/PadNoteTest.h
+++ b/src/Tests/PadNoteTest.h
@@ -167,7 +167,7 @@ class PadNoteTest:public CxxTest::TestSuite
 
             note->noteout(outL, outR);
             sampleCount += synth->buffersize;
-            TS_ASSERT_DELTA(outL[255], 0.0378f, 0.0001f);
+            TS_ASSERT_DELTA(outL[255], 0.0378f, 0.0005f);
 
             note->noteout(outL, outR);
             sampleCount += synth->buffersize;
-- 
2.47.0

